packages/pouchdb-ajax/src/request-browser.js

```diff
@@ -126,8 +126,7 @@ function xhRequest(options, callback) {
   try {
     xhr.open(options.method, options.url);
   } catch (exception) {
-   /* error code hardcoded to throw INVALID_URL */
-    callback(exception, {statusCode: 413});
+    return callback(new Error(exception.name || 'Url is invalid'));
```

Comment:
So this was a fairly big thing to miss and hopefully it may weed out some fairly confusing behaviour, its likely the source of the need for extra tests + istanbul comments needed to not regress coverage